bool clear_arrows
geometry_msgs/PoseStamped[] arrows
